lib/v8-to-istanbul.js

```diff
@@ -51,7 +52,30 @@ module.exports = class V8ToIstanbul {
     if (this.rawSourceMap) {
       if (this.rawSourceMap.sourcemap.sources.length > 1) {
         this.sourceMap = await new SourceMapConsumer(this.rawSourceMap.sourcemap)
-        this.covSources = this.sourceMap.sourcesContent.map((rawSource, i) => ({ source: new CovSource(rawSource, this.wrapperLength), path: this.sourceMap.sources[i] }))
+        // https://github.com/istanbuljs/v8-to-istanbul/issues/166
+        if (!this.sourceMap.sourcesContent) {
+          this.sourceMap.sourcesContent = []
+          for (const relativePath of this.sourceMap.sources) {
+            const fileDir = path.resolve(this.path, '..')
+            const realPath = path.resolve(fileDir, relativePath)
+            let rawSource = null
+            try {
+              rawSource = await readFile(realPath, 'utf-8')
+            } catch (err) {
+              // Perhaps some libraries do not provide source files.
+            }
+            this.sourceMap.sourcesContent.push(rawSource)
```

Comment:
do we want to push `null`?

Reply:
yep

Reply:
em, perhaps it would be better to remove `null`, I will modify that.